com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ApplicationStager.java

```diff
@@ -91,10 +91,12 @@ public StepPhase stageApp(DelegateExecution context, CloudControllerClient clien
         UUID packageGuid = uploadToken.getPackageGuid();
 
         stepLogger.info(Messages.STAGING_APP, app.getName());
-
-        context.setVariable(Constants.VAR_BUILD_GUID, client.createBuild(packageGuid)
-            .getMetadata()
-            .getGuid());
+        CloudBuild build = client.createBuild(packageGuid);
+        UUID buildGuid = build.getMetadata()
+                              .getGuid();
+        stepLogger.getProcessLogger()
+                  .info(MessageFormat.format(Messages.BUILD_CREATED, buildGuid));
```

Comment:
Is there any particular reason that this is an info?
Would the debug also apply in this case?
If you are worried that the debug would be printed on the client side - dont be. The client only prints the INFO progress messages.